Update Deployment Settings and hosted reference docs (#3317)
astronomer/docs: astro/deployment-settings.md

---
sidebar_label: 'Overview'
title: 'Overview'
id: deployment-settings
description: "Manage Deployment settings to customize your Deployment for your use case."
---

> After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its performance. There are two categories of options for configuring your Astro Deployment to best meet your needs: Deployment details and resource configuration.

Reviewer: ```suggestion
After you create an Astro Deployment, you can modify its settings using the Cloud UI and Astro CLI to tailor its performance. There are two categories of Deployment configurations: Deployment _details_ and _resources_.
```

## Deployment details

Use [Deployment details](deployment-details.md) to manage the administration of your Deployment. This includes includes actions like updating information about your Deployment or transferring it to another Workspace. The following actions allow you to organize and label your Deployments within your Workspace in a way that best fits how your team works with Astro:

> - Update a Deployment name and description.
> - Add or delete a Deployment alert email.

Reviewer: ```suggestion
- Deployment names and descriptions.
- Deployment contact emails.
- Rules for what types of deploys are allowed.
```
It felt odd to present a list of verbs to describe the noun of "details", so framing this around objects in stead of actions

> Use [Deployment details](deployment-details.md) to manage the administration of your Deployment. This includes includes actions like updating information about your Deployment or transferring it to another Workspace. The following actions allow you to organize and label your Deployments within your Workspace in a way that best fits how your team works with Astro:

Reviewer: ```suggestion 
[Deployment details](deployment-details.md) are high level settings that define how users can interact with the Deployment. These settings include:
```

> - Transfer a Deployment to another Workspace in your Organization.

Reviewer: ```suggestion
```
This isn't included in the details doc atm

> - Rules for what types of deploys are allowed.

Reviewer: ```suggestion
- Rules for what types of deploys are allowed.
- Your Deployment's current Workspace.
```

> - Delete a Deployment.

Reviewer: ```suggestion
```

Reviewer: Let's cut this for not since it's kind of the odd one out and might be refactored in the future

Reviewer: Or frame it as a noun somehow

## Deployment resources

> You also have [configuration options](deployment-resources.md) that allow you to customize the resource use, infrastructure, and performance of your Deployment. This means you can optimize how your Deployment uses resources, so you can improve the efficiency of your Deployment processing, optimize compute resources and cost, or enable advanced use cases with intensive workloads. The following configuration options enable you to customize your Deployment to get the most out of your resources on Astro:

Reviewer: ```suggestion
[Deployment resources](deployment-resources.md) let you customize the resource use, infrastructure, and performance of your Deployment. Use these settings to optimize Deployment processing, optimize compute resources and cost, and enable advanced use cases with intensive workloads. The following configuration options are available for customizing your resource use:
```

- Change the Deployment executor.

> - Configure Kubernetes Pod resources.

Reviewer: There's a bullet in here called "Update your Airflow configuration". Maybe we should make this more specific as: "Customize your Airflow environment through environment variables"

> This document focuses on configuring Deployments through the Cloud UI. To configure Deployments as code using the Astro CLI, see [Manage Deployments as code](manage-deployments-as-code.md).

Reviewer: ```suggestion
These documents focus on configuring Deployments through the Cloud UI. To configure Deployments as code using the Astro CLI, see [Manage Deployments as code](manage-deployments-as-code.md).
```
